lms/djangoapps/discussion_api/tests/test_serializers.py

```diff
@@ -440,3 +441,107 @@ def test_type(self):
         data["type"] = "invalid_type"
         serializer = ThreadSerializer(data=data)
         self.assertFalse(serializer.is_valid())
+
+
+@ddt.ddt
+class CommentSerializerDeserializationTest(CommentsServiceMockMixin, ModuleStoreTestCase):
+    """Tests for ThreadSerializer deserialization."""
+    def setUp(self):
+        super(CommentSerializerDeserializationTest, self).setUp()
+        httpretty.reset()
+        httpretty.enable()
+        self.addCleanup(httpretty.disable)
+        self.course = CourseFactory.create()
+        self.user = UserFactory.create()
+        self.register_get_user_response(self.user)
+        self.request = RequestFactory().get("/dummy")
+        self.request.user = self.user
+        self.minimal_data = {
+            "thread_id": "test_thread",
+            "raw_body": "Test body",
+        }
+
+    def save_and_reserialize(self, data, parent_id=None):
```

Comment:
The input is serialized, so this is a re-serialization of the data after it has been saved.